gpbft/api.go

```diff
@@ -54,9 +54,10 @@ type MessageReceiver interface {
 
 // Interface which network participants must implement.
 type Receiver interface {
-	// Begins executing the protocol.
-	// The node will request the canonical chain to propose from the host.
-	Start() error
+	// SkipToInstance jumps directly to a given instance.
+	// This can be triggered by the reception of a valid finality certificate, or
+	// whenever a new instance for a participant want to be started.
+	SkipToInstance(uint64)
```

Comment:
I didn't mean for you to actually remove Start from the interface here, just that we should consider it after this change.

If we're going to remove it, we should do so more thoroughly by removing it from Participant and all the adversary participants, and probably removing the `initialInstance` config parameter. Rather than do that now, let's put it back (still async) and follow up to remove it.